<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>681</width>
      <height>424</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPlainTextEdit" name="teOutput">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listPlaylist">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listSong"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWeb_API">
    <property name="title">
     <string>Web API</string>
    </property>
    <addaction name="actionGrant"/>
    <addaction name="actionGet_Playlists"/>
    <addaction name="actionCreate_Playlist"/>
    <addaction name="actionGet_User_Information"/>
   </widget>
   <addaction name="menuWeb_API"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGrant">
   <property name="text">
    <string>Grant</string>
   </property>
  </action>
  <action name="actionGet_Playlists">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get Playlists</string>
   </property>
   <property name="toolTip">
    <string>Query Playlists from Webapi</string>
   </property>
  </action>
  <action name="actionGet_User_Information">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get User Information</string>
   </property>
  </action>
  <action name="actionCreate_Playlist">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Playlist</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>счябо</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
